src/components/typography/Heading/Heading.styles.js

```diff
@@ -23,8 +23,10 @@ export const headingColors = {
 };
 
 const HeadingBase = styled.span`
-  color: ${p => headingColors[p.color]};
-  ${p => headingSizes[p.size]};
+  margin: ${props => props.margin};
```

Comment:
I don't see where `props` are defined here. Do you have access to `props` here via some magic when importing this?

Reply:
Yes? I'm not sure; it was just there in the file when I forked it!